src/portable/portable.c

```diff
@@ -1025,7 +1026,7 @@ static int install_chroot_dropin(
                                         return -ENOMEM;
         }
 
-        r = write_string_file(dropin, text, WRITE_STRING_FILE_CREATE|WRITE_STRING_FILE_ATOMIC);
+        r = write_string_file(dropin, text, WRITE_STRING_FILE_CREATE|WRITE_STRING_FILE_ATOMIC|WRITE_STRING_FILE_ENABLE_FS_VERITY);
```

Comment:
does this make sense? this is not copied out data, it's dynamically generated configuraiton. why would fsverity get you anything there?